### ISSUE TITLE
Automatic escape special characters

### ISSUE THREAD

**Issue by milianriedel** [blueberryapps/translation-server#52]
In translation tile`cs.menu_link:  Risk & Fraud Management`

After save translation in translation server transform to  `Risk &amp; Fraud Management`

@ondrejbartas ?

**Comment by ondrejbartas** [comment]
nope, not good idea. If you want to do any transformations like this, we need to do that in place where we insert data...
